src/diy_set.md

```diff
@@ -0,0 +1,109 @@
+### Box contents
+Inside your SlimeVR DIY kit are 7 large SlimeVR Mainboards, 3 small SlimeVR Tracker extensions, and 3 Extension cables (1 long, 2 short).
+
+<center><img id="ContentImage" src="./assets/img/DIY_KIT_box-contents.jpg" alt="Box contents image" style="height:400px"/></center>
+
+
+### Required extras
+To complete your SlimeVR trackers, you will need some parts in addition to your SlimeVR DIY kit. These include enclosures, straps, batteries, and a USB cable. These are detailed below to make it easier for you to complete your project.
+
+### Recommended mounting layout
+TODO: Currently placeholder image
+
+<table class="bpTable">
+   <tr>
+      <td>
+         <img id="bpImage" src="./assets/img/DIY_KIT_tracker-locations.png" alt="Mounting locations image" style="height:500px"/>
+      </td>
+      <td>
+         <ol>
+            <li>Chest (Extension)</li>
+            <li>Waist</li>
+            <li>Upper legs</li>
+            <li>Lower legs</li>
+            <li>Feet (Extension)</li>
+            <li>Upper arms</li>
+         </ol>
+      </td>
+   </tr>
+</table>
+
+## Designing your case
+DIY kits do not come with cases and such require enclosures to protect them and allow strap attachment. These can be printed, purchased, or crafted. While the official SlimeVR Main and Extension cases are designed to fit the boards included in this kit, they are not optimised for home 3d printers due to their curved surfaces. As such, we recommend printing up your own using a modified design, or designing your own!
+
+The 3D models for the official main and extension cases can be found [here](TODO:linktofiles). Modified versions of these official cases more suited for home 3D printers, made by Tom Yum#0069, can be found [here](TODO:linktoprintables?). 
+
+For those planning to design your own enclosure, you can find the model of the Mainboard [here](TODO:linktomainboardstepfile) and Extension PCBs [here](TODO:linktoextensionstepfile).
+
+<details>
+<summary><u>Using the official case designs?</u></summary>
+<br>
+TODO: assembly guide
+</details>
+<br>
+
+## Choosing your battery
+The SlimeVR Mainboards included in your DIY kit require power from a battery for the best experience. SlimeVR trackers use approximately 100mAh during use. When choosing a battery, **select a capacity rating of at least 500mAh to ensure safe use.** Charging rate is limited to 500mA to ensure optimal battery lifespan. 
+
+We recommend the following specifications:
+* Capacity: 1000-1800 mAh
+* Nominal Voltage: 3.7v
+* Connection: Micro JST 1.25mm
+
+The battery dimensions will depend on which case you choose to use or make. Lithium-ion Polymer (LiPo) pouch batteries come in various shapes and sizes, indicated by an `XXYYZZ` naming scheme that denotes their dimensions in thickness (X.Xmm), width (YYmm), and length (ZZmm). As an example, the official SlimeVR case design uses an 803443 battery, denoting a battery of 8.0mm thickness, 34mm width, and 43mm length.
```

Comment:
Can add that it can fit 804040 or 604040 too.

Reply:
yeh I wanted to add the dimensions of the cavity and a few examples so this info is good ty!